gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcheckboxlabel_ttt2.lua

```diff
@@ -63,6 +63,13 @@ function PANEL:Paint(w, h)
 	return true
 end
 
+---
+-- @param bool invert
+-- @realm client
+function PANEL:SetInverted(invert)
+	self.inverted = tobool(invert)
+end
```

Comment:
you define the param as a bool, but still parse it?

Reply:
Yeah as there are no types, I just make sure its the type I want. I could probably leave it away. Gotta check it.